module/x/gravity/types/genesis.go

```diff
@@ -65,6 +67,10 @@ var (
 	//  ParamStoreUnbondSlashingSignerSetTxsWindow stores unbond slashing valset window
 	ParamStoreUnbondSlashingSignerSetTxsWindow = []byte("UnbondSlashingSignerSetTxsWindow")
 
+	// ParamStoreEthereumBlacklist allows storage of blocked Ethereum addresses blocked for use with the bridge
+	// this could be for technical reasons (zero address) or non-technical reasons, these apply across all ERC20 tokens
```

Comment:
Here it mentioned zero address is blocked for technical reason. Perhaps it would be good to specify what the reason is or at least point to some resource.

Reply:
I removed the comment